Helper/Email/PendingOrderEmailConfirmation.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Paytrail\PaymentService\Helper\Email;
```

Comment:
Helpers are considered anti pattern in modern Magento 2.
More information here: https://developer.adobe.com/commerce/php/best-practices/extensions/#avoid-creating-helper-classes

Please take a moment to consider the structure of your code and think of a location for your code that has an accurate namespace and reflects the role of the code in this class. Also take a moment to think if all the code in this class should be in it after the refactoring.

For modern Magento we should try to refactor and remove existing helpers. Instead of creating new ones or updating old ones.

Reply:
Changed Helper into Model